### DISCUSSION TITLE
STELLA 1.1 Air Temp

### DISCUSSION THREAD

**Discussion by Landsat-Mike** [STELLA-Landsat/STELLA#23] [Debugging]
The Air temp says no fix on the screen. I ran the test code and it is getting a temperature reading. I also checked the connections with my multimeter so I am not sure what's happening.

**Comment by paulmirel** [comment]
That was an earlier versions of the S1.1 software, where I was working out how to show GPS status. With the current version, that message "no fix" does not display when the GPS unit is not connected. 

(I have yet to get the GPS fully working, so as of 2023-06-12 it is not yet operable as an add on)
